keyboards/dz60/keymaps/nazu/rules.mk

```
@@ -0,0 +1,17 @@
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no	# Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes	# Audio control and System control(+450)
+CONSOLE_ENABLE = no		# Console for debug(+400)
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+AUTO_SHIFT_ENABLE = no  # If the time depressed is greater than or equal to the AUTO_SHIFT_TIMEOUT, then a shifted version of the key is emitted. If the time is less than the AUTO_SHIFT_TIMEOUT time, then the normal state is emitted
+TAP_DANCE_ENABLE = yes
+LTO_ENABLE = yes 	    # significantly reduce the compiled size, but disable the legacy TMK Macros and Functions features
+VIA_ENABLE = yes
```

This can be cleaned up.  Eg, it only really needs what is different from the keyboard's settings. Which appears to be: 

```suggestion
MOUSEKEY_ENABLE = no

AUTO_SHIFT_ENABLE = no 
TAP_DANCE_ENABLE = yes
LTO_ENABLE = yes 
VIA_ENABLE = yes
```